packages/react/src/components/DataTable/TableToolbarSearch.js

```diff
@@ -11,19 +11,15 @@ import React, { useRef, useState, useEffect } from 'react';
 import { settings } from 'carbon-components';
 import Search from '../Search';
 import setupGetInstanceId from './tools/instanceId';
+import deprecate from '../../prop-types/deprecate';
 
 const { prefix } = settings;
-
 const getInstanceId = setupGetInstanceId();
 const translationKeys = {
   'carbon.table.toolbar.search.label': 'Filter table',
   'carbon.table.toolbar.search.placeholder': 'Search',
 };
-
-const translateWithId = id => {
-  return translationKeys[id];
-};
-
+const translateWithId = id => translationKeys[id];
```

Comment:
As a code style nit, I tend to prefer the expanded form because debugging is easier. In the case of the implicit return, one must unwrap the block to the explicit return syntax and then convert it back when done debugging. Meaning:

```js
// 1 Original
const translateWithId = id => translationKeys[id];

// 2 Unwrap to debug
const translateWithId = id => {
  console.log(id);
  return translationKeys[id]
}; 

// 3, transform it back
const translateWithId = id => translationKeys[id];
```

Feels like if we're optimizing for maintainability that we should just keep form 2

Reply:
> one must unwrap the block to the explicit return syntax and then convert it back when done debugging

you can `console.log` debug with

```js
const translateWithid = id => console.log(id) || translationKeys[id];
```

but I reverted the change anyway